examples/vqgan/README.md

```diff
@@ -0,0 +1,119 @@
+## Training an VQGAN VAE
```

Comment:
Maybe add a few lines about what the training script is about?

Reply:
Do you think the brief explanation of a VQGAN I made is needed here?

Reply:
The current one works :)